### DISCUSSION TITLE
Fold and unfold since version 1.5.1

### DISCUSSION THREAD

**Discussion by itzikd15** [ajaxorg/ace#5065] [Q&A] [answered]
Hello, I am trying to upgrade from version 1.5.1 but since version 1.5.2 and forward i cant use the session.unfold() / session.foldAll()
<img width="302" alt="image" src="https://user-images.githubusercontent.com/111684574/220016375-4b6a880e-33f4-46ee-9c2f-9c4011f46cf9.png">

i can see in the changes over here:
https://github.com/ajaxorg/ace/commit/d5d6f9a32ad443d02aa89d9f140917637f5f52ab

that editSession doesn't support folding any more
<img width="775" alt="image" src="https://user-images.githubusercontent.com/111684574/220016522-11b9d6bd-0dcb-4e67-9d4a-ac1884c7be59.png">

how can i fold all the lines in my code and also how can i unfold?

thanks!

**Comment by akoreman** [comment, marked as answer]
The folding interface was restored in version [1.15.3](https://github.com/ajaxorg/ace/commit/cc98bdcb76a1a24cef3997af52d78c4880590508).
